<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothWidget</class>
 <widget class="QWidget" name="BluetoothWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DevicesBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="DisconnectButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ScanButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoadingGif">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>LoadingGif</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
